condor/migration-guide.md

```diff
@@ -0,0 +1,112 @@
+
+### RPC Integration
+Casper 2.0 implements a number of changes to the JSON RPC interface of the node, which includes new methods, renamed methods and methods with a change in response format. 
+Importantly, the block structure is different in Casper 2.0 vs. Casper 1.x, and `Deploys` have been renamed to `Transactions`. 
+
+At the same time, legacy (Casper 1.x) deploys will continue to be accepted by Casper 2.0, hence if your entire integration comprises only the submission of Deploys (such as transfers in and out of accounts), you may not have to make any changes prior to mainnet activation of Casper 2.0. 
+
+➡️ To assess your next steps, please evaluate your integration agains the 1.x -> 2.0 JSON RPC changes detailed in this document. 
+
+### SDK Integration
+Depending on which SDK language (e.g. JS, Go) you use, and which methods you interface with, you might not have to immediately migrate your integration. However, in order to be safe and future-proof, it is strongly recommended to upgrade your SDK to the latest version applicable to your stack. For most languages, this can be done immediately, as the SDKs maintain both forward (2.x) and backward (1.x) compatibility at the same time. 
+
+Please see instructions below for your specific SDK language:
+
+| SDK    | Instructions | 
+|---------------------|-------------|
+| **JavaScript** | While it may not be strictly necessary to upgrade your integration if all you do is submit `Deploys` to the network (e.g. transfers in and out of accounts), it is STRONGLY RECOMMENDED to upgrade to the new SDK now, to experience a seamless upgrade to Casper 2.0 and beyond. |
+| |The appropriate SDK to upgrade to is the lates v5.x release found [here](https://github.com/casper-ecosystem/casper-js-sdk/releases). |
+| | Common use case examples (connect to RPC, send a transfer, etc.) can be found in [Casper JS SDK 5.0 README](https://github.com/ casper-ecosystem/casper-js-sdk/tree/feat-5.0.0?tab=readme-ov-file#usage-examples) |
+| |To upgrade your integration, follow the detailed [V2 to V5 Migration Guide](https://github.com/casper-ecosystem/casper-js-sdk/blob/feat-5.0.0/resources/migration-guide-v2-v5.md) | 
+| **#C .NET** / **Go** / **Java** | While it may not be strictly necessary to upgrade your integration if all you do is submit `Deploys` to the network (e.g. transfers in and out of accounts), it is STRONGLY RECOMMENDED to upgrade to the new SDK now, to experience a seamless upgrade to Casper 2.0 and beyond.|
+| **Other** | If you are using an SDK not listed in this chapter, please contact the Casper Exchange Support Team.|
```

Comment:
@devendran-m 
Content for Go SDK:

- The appropriate SDK to upgrade to is the latest v2.x release found [[here](https://github.com/make-software/casper-go-sdk/releases)](https://github.com/make-software/casper-go-sdk/releases).
- Typical use case examples (connect to RPC, send a transfer, etc.) can be found in the [[README](https://github.com/make-software/casper-go-sdk/blob/master/README.md)](https://github.com/make-software/casper-go-sdk/blob/master/README.md).

Reply:
Thanks @davidatwhiletrue , 

I've incorporated your comments in the commit - [ff82e88](https://github.com/casper-network/docs-redux/pull/5/commits/ff82e885077f753148d455964421e05cf0044494)

Please review and let me know in case of any other changes. 

cc: @KMCreatesWorlds